### PR TITLE
feat(wallet): Wallet faq

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves
Implements the generic FAQ section with content imported from the JSON file. Similar to https://github.com/safe-global/safe-homepage/tree/main/src/components/commonCMS/Faq

## Figma
https://www.figma.com/design/qtXXlwdslAze6yNbLLzIRa/Website-redesign-2.0?node-id=15367-15363&node-type=frame&t=qN9WsCrCOsfBan3X-0

Screenshot 2024-11-04 at 16 48 13

### CODE REVIEW

File: src/content/wallet.json [hunk 1]
```diff
@@ -211,6 +211,32 @@
       }
     ]
   },
+  {
+    "component": "common/FAQ",
+    "title": "FAQ",
+    "items": [
+      {
+        "title": "What is a Safe smart account?",
```
Comment (iamacook) [inline src/content/wallet.json:219]:
```suggestion
        "title": "What is a Safe Smart Account?",
```

File: src/content/wallet.json [hunk 2]
```diff
@@ -211,6 +211,32 @@
...
+        "title": "Can I use Safe{Wallet} for personal funds?",
+        "text": "Yes, Safe{Wallet} can be used for personal funds. It provides a secure environment for managing your crypto assets, whether for personal or business use."
+      },
+      {
+        "title": "How do I create an account?",
+        "text": "To create an account with Safe{Wallet}, all you need is a wallet address. You'll use this address to create your Safe Smart Account, which is a multisig account. You'll then deploy a smart contract to manage your funds securely. The process is simple and guided within the Safe{Wallet} platform."
+      },
+      {
+        "title": "What networks are supported by Safe{Wallet}?",
+        "text": "Safe{Wallet} supports a wide range of blockchain networks, allowing you to manage your crypto assets across multiple platforms. The supported networks include Ethereum, Gnosis Chain, Polygon, Polygon zkEVM, BNB Chain, Arbitrum, Optimism, Base, Linea, zkSync Era, Scroll, X Layer, Celo, Avalanche, and Aurora. You can easily select the appropriate network when setting up your Safe Smart Account or when managing your funds within the Safe{Wallet} platform."
```
Comment (iamacook) [inline src/content/wallet.json:236]: What do you think about populating this list from the Config Service?

File: src/components/common/FAQ/index.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                onChange={handleChange}
+                key={index}
+                disableGutters
+                square
+              >
+                <AccordionSummary
+                  expandIcon={expanded ? <MinusIcon /> : <PlusIcon />}
+                  onClick={() => {
+                    !expanded
+                  }}
```
Comment (iamacook) [inline src/components/common/FAQ/index.tsx:53]:
```suggestion
```

File: src/components/common/FAQ/index.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                [index]: expanded,
+              }))
+            }
+            const expanded = openMap?.[index] ?? false
+
+            return (
+              <Accordion
+                className={css.accordion}
+                expanded={expanded}
+                onChange={handleChange}
```
Comment (iamacook) [inline src/components/common/FAQ/index.tsx:44]: Why is this being controlled? The `Accordion` [doesn't need to be](https://mui.com/material-ui/react-accordion/#controlled-accordion).

File: src/content/wallet.json (hunk 2 above)
Comment (DiogoSoaress) [reply src/content/wallet.json:236]: I would rather find a solution through the CMS. Let me iterate on it and will ping you again to review.

File: src/components/common/FAQ/index.tsx (hunk 4 above)
Comment (DiogoSoaress) [reply src/components/common/FAQ/index.tsx:44]: It was the way I found to `setOpenMap` and control which accordions are open by index.

File: src/content/wallet.json (hunk 1 above)
Comment (DiogoSoaress) [reply src/content/wallet.json]: Applied in the CMS entry. You can verify in the PR deployment.

**Review by iamacook — APPROVED** [review]

**Pull request merged**